packages/formik/test/FieldArray.test.tsx

```diff
@@ -348,11 +349,29 @@ describe('<FieldArray />', () => {
         </TestForm>,
         node
       );
-
+    });
+    it('should remove a value at given index of field array', () => {
       arrayHelpers.remove(1);
       const expected = ['jared', 'brent'];
       expect(formikBag.values.friends).toEqual(expected);
     });
+
+    it('should be an empty array when removing all values', () => {
+      arrayHelpers.remove(0);
```

Comment:
seems weird calling 0 multiple times, but every time we call remove, the indexes get updated.

Reply:
Add this in a comment